tests/Envs/IntegerBackedEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crell\EnvMapper\Envs;
+
+enum IntegerBackedEnum: int {
+    case FOO = 1;
+    case BAR = 2;
+    case BAZ = 3;
```

Comment:
Enum cases should not be in all caps.  They should be CamelCase.

Reply:
Fixed.